### PR TITLE
Sort the generated CodeAction telemetry table.

### PR DESCRIPTION
**Pull request by JoeRobich**
This change should make future updates easier to validate.

### CODE REVIEW

File: src/Tools/BuildActionTelemetryTable/Program.cs [hunk 1]
```diff
@@ -25,321 +25,333 @@ public class Program
...
+            "Microsoft.CodeAnalysis.CodeActions.CodeAction+SolutionChangeAction",
             "Microsoft.CodeAnalysis.CodeActions.CustomCodeActions+DocumentChangeAction",
             "Microsoft.CodeAnalysis.CodeActions.CustomCodeActions+SolutionChangeAction",
             "Microsoft.CodeAnalysis.CodeStyle.AbstractCodeStyleProvider`2+CodeRefactoringProvider",
+            "Microsoft.CodeAnalysis.CodeFixes.DocumentBasedFixAllProvider+PostProcessCodeAction",
         }.ToImmutableHashSet();
 
         private static ImmutableDictionary<string, string> CodeActionDescriptionMap { get; } = new Dictionary<string, string>()
         {
+            { "Microsoft.CodeAnalysis.AddConstructorParametersFromMembers.AddConstructorParametersFromMembersCodeRefactoringProvider", "Add Constructor Parameters From Members (Refactoring)" },
```
Comment (JoeRobich) [inline src/Tools/BuildActionTelemetryTable/Program.cs:40]: This table was regenerate after sorting was added.

File: src/Tools/BuildActionTelemetryTable/Program.cs [hunk 2]
```diff
@@ -498,6 +498,7 @@ internal static ImmutableArray<TelemetryInfo> GetTelemetryInfos(ImmutableArray<T
             return codeActionAndProviderTypes
                 .Distinct(FullNameTypeComparer.Instance)
                 .Select(GetTelemetryInfo)
+                .OrderBy(info => info.Item1)
```
Comment (JoeRobich) [inline src/Tools/BuildActionTelemetryTable/Program.cs:501]: This is the only functional change which sorts based on TypeName.

**Review by genlu — APPROVED** [review]
How does this list of actions got validated?

**Comment by JoeRobich** [comment]
> How does this list of actions got validated?

Telemetry for CodeActions is reported by either CodeAction name or in the case of using `CodeAction.Create` the Provider name. Mostly I meant that when we regenerate the mapping table it would be easier to read the diff if things are output in a consistent order.

**Review by CyrusNajmabadi — APPROVED** [review]

File: src/Tools/BuildActionTelemetryTable/Program.cs (hunk 2 above)
Comment (CyrusNajmabadi) [reply src/Tools/BuildActionTelemetryTable/Program.cs:501]: nit, consdier named tuple fields to make this clearer.

**Pull request merged**